modules/oxide/src/less/theme/content/print/print.less

```diff
@@ -16,5 +15,9 @@
     .mce-edit-focus {
       outline: none !important;
     }
+
+    tiny-uploadcare-placeholder {
+      display: none !important; 
+    }
```

Comment:
This might be better in `uploadcare.less`? The other plugins that have print-specific styles are in their individual less files 🤔 
https://github.com/tinymce/tinymce/blob/236891ffcdf2fd1b698d020e67c31838fb110019/modules/oxide/src/less/theme/content/pagebreak/pagebreak.less#L15-L19